docs/ui/installation.rst

```diff
@@ -34,13 +34,27 @@ The sample configuration makes the UI load job/push data from the production ser
 
 If you wish to run the full treeherder Vagrant project (service + UI), remember to remove local.conf.js or else change ``thServiceDomain`` within it to refer to ``vagrant``, so the UI will use the local Vagrant service API.
 
+Validating JavaScript
+=====================
+
+We run our JavaScript code in the frontend through eslint_ to ensure
+that new code doesn't suffer from common errors. Before submitting a
+patch, check that your code passes these tests
```

Comment:
Dunno perhaps a trailing period here, it's seems to be the paragraph style for this file.